net/Socket.cpp

```diff
@@ -1517,31 +1517,28 @@ bool StreamSocket::checkRemoval(std::chrono::steady_clock::time_point now)
                     << *this);
             if (_socketHandler)
             {
-                _socketHandler->onDisconnect();
-                if( isOpen() ) {
-                    // Note: Ensure proper semantics of onDisconnect()
-                    LOG_WRN("Socket still open post onDisconnect(), forced shutdown.");
-                    shutdown(); // signal
-                    closeConnection(); // real -> setClosed()
+                _socketHandler->onDisconnect(); // Note: Ensure proper semantics of onDisconnect()
+                if( isOpen() )
```

Comment:
Spacing could also benefit from fixing up: `if (isOpen())`.

The reason this is more readable (I think) is because language statements are (emphatically) not function calls. Making them stand out helps the eye scan faster and more accurately. I suspect the preference to make them look like function calls is to homogenize the code, but in this case it can be misleading to the reader. Ditto for `while`, `for`, `do`, `return`, etc.